README.md

```diff
@@ -0,0 +1,33 @@
+Requests for PHP Test Server
+============================
+
+This repository is a simplistic test server that is being used by the [Requests for PHP library](https://github.com/WordPress/Requests/) to run its automated testing suite.
+
+Installation
+------------
+
+This package is not meant to be used as a stand-alone server. Rather, it is being used by the CI testing workflows in the [Requests for PHP library](https://github.com/WordPress/Requests/).
+
+To install it in your local environment anyway, you can use the following command:
+
+```bash
+composer install
+```
+
+Note: This package requires **Composer v2.2+** to work correctly.
+
+Contribute
+----------
+
+1. Check for open issues or open a new issue for a feature request or a bug.
+2. Fork [the repository][] on Github to start making your changes to the
```

Comment:
url is missing here

Reply:
This is being taken from the URLs below. This is a special notation to split the URLs out for the links as footnotes.